README.md

```diff
@@ -1 +1,67 @@
-# Beyond-07-team-4
\ No newline at end of file
+# Job Seeker
+
+**Job Seeker** is a web platform, built for people to post and find jobs.
+It might serve students, freelancers or organzations.
+
+---
+
+## 🏃 How To Run The Project
+
+### Prerequisites
+
+You should install the following programs before proceeding:
+
+- #### Git (*Advised For Downloading This Project*)
+    > Git is a Version Control System (VCS) - the practice of tracking and managing changes to software code. Version control systems are software tools that help software teams manage changes to source code over time.
+
+    We will use Git to download this project in the steps below.
+    [Download Git Here!](https://git-scm.com/downloads)
+
+- #### VirtualBox
+    > VirtualBox is a hypervisor, which lets us create and run virtual machines.
+
+    During the installation process, Windows might pop-up some warning windows about installing device driver software, please confirm for it that the software should be installed.
+    VirtualBox supports various OS to host virtual machines.
+    [Download VirtualBox Here!](https://www.virtualbox.org/wiki/Downloads)
+
+- #### Vagrant
+    > Vagrant is a tool for using virtual machines to share development environments.
+    > Learn more about it [here](https://www.vagrantup.com/).
+    
+    Please make sure you have Vagrant installed on your computer, if not then install the latest version of Vagrant for your current operating system.
+    [Download Vagrant Here!](https://www.vagrantup.com/downloads)
+
+#### **ATTENTION:**
+In order for it all to work, please enable **virtualization hardware** on your computer. An explanation on how to do it can be found [here](https://www.virtualmetric.com/blog/how-to-enable-hardware-virtualization).
+
+---
+
+### Run The Project
+
+#### Download The Project
+
+On your computer:
+1. Navigate to the folder you wish to place the project at.
```

Comment:
Navigate the folder you wish to place the project at.

Reply:
Please remove it, 
just the git clone command line and that's it

Reply:
@rrasouli @kasemAlem changes were made in [72b2282](https://github.com/redhat-beyond/JobSeeker/pull/13/commits/72b228281fe69200fadfdf7e3b863e3116002644), please review.